src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private static final double DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = String.valueOf(Color.WHITE);
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE.name();
```
jv-lottery common mistakes
![image](https://github.com/mate-academy/jv-oop-advanced/assets/49767049/82cef5bc-aa7e-432b-a766-ab9c2e49b0f5)